example/didi/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM sqlflow/sqlflow:latest
+# Prepare sample datasets
+COPY  titanic/titanic.sql   carprice/carprice.sql   /docker-entrypoint-initdb.d/
```

Comment:
I have not tried to build this Docker image, but maybe we need to add  the `ipynb` tutorials into this Docker image by adding the following context:
``` text
...
ADD example/didi/carprice/tutorial_xgboost_carprice.ipynb /workspace/
ADD example/didi/titanic/tutorial_dnn_titanic.ipynb /workspace/
CMD ["bash", "/start.sh"]
```

Reply:
Because the Dockerfile in the didi folder, maybe we adding the following context instead?
ADD carprice/tutorial_xgboost_carprice.ipynb /workspace/
ADD titanic/tutorial_dnn_titanic.ipynb /workspace/
CMD ["bash", "/start.sh"]

Reply:
Yes, you're right.